app/controllers/nollning/nollnings_controller.rb

```diff
@@ -7,7 +7,13 @@ def index
   end
 
   def matrix
-    @events = Event.includes(:translations).slug(:nollning)
+    @dates = Date.new(2016, 8, 22)..Date.new(2016, 9, 23)
```

Comment:
I was thinking of creating a whole `Nollning`-model but decided not to.
However that might actually be wort it - could be cool for history as well.